backend/workflow_manager/endpoint/destination.py

```diff
@@ -409,3 +415,42 @@ def get_json_schema_for_api(cls) -> dict[str, Any]:
             os.path.dirname(__file__), "static", "dest", "api.json"
         )
         return cls.get_json_schema(file_path=schema_path)
+
+    def _push_to_queue(
+        self,
+        file_name: str,
+        error: Optional[str] = None,
+        result: Optional[str] = None,
+    ) -> None:
+        """Handle the QUEUE result.
+
+        This method is responsible for pushing the input file and result to
+        review queue.
+        Args:
+            file_name (str): The name of the file.
+            result (Optional[str], optional): The result of the API call.
+                Defaults to None.
+
+        Returns:
+            None
+        """
+        connector: ConnectorInstance = self.endpoint.connector_instance
+        connector_settings: dict[str, Any] = connector.connector_metadata
+        queue_result = {"file": file_name}
+        if error:
+            return
+        if result:
```

Comment:
@vishnuszipstack this can be refactored as 
```
if not result:
    return
```
This will help reduce nesting of the code.
Another option is to forcefully pass a `result` into this function and perform a check generically at the caller side

Reply:
done @chandrasekharan-zipstack